SwiftWisdom/Core/Downloader/Downloader.swift

```diff
@@ -82,3 +91,9 @@ extension NSURLSessionTask {
         return min(percentage, 1)
     }
 }
+
+extension NSHTTPURLResponse {
+    public var isSuccess: Bool {
+        return 200 ... 299 ~= statusCode
```

Comment:
wow. i really learned something new here.

Also, tiny note, remove the spaces between the numbers and ...